pkg/storage/snapshot/restore.go

```diff
@@ -51,6 +50,7 @@ import (
 	backendstorage "kubevirt.io/kubevirt/pkg/storage/backend-storage"
 	typesutil "kubevirt.io/kubevirt/pkg/storage/types"
 	storageutils "kubevirt.io/kubevirt/pkg/storage/utils"
+	naming "kubevirt.io/kubevirt/pkg/util/naming"
```

Comment:
```suggestion
	"kubevirt.io/kubevirt/pkg/util/naming"
```

Should work the same

Reply:
done